app/models/post.js

```diff
@@ -180,10 +180,11 @@ export default Model.extend(Comparable, ValidationEngine, {
         return this.get('ghostPaths.url').join(blogUrl, postUrl);
     }),
 
-    previewUrl: computed('uuid', 'ghostPaths.url', 'config.{blogUrl,routeKeywords.preview}', function () {
+    previewUrl: computed('uuid', 'ghostPaths.url', function () {
```

Comment:
Minor thing but we still use `config.blogUrl` inside this CP so we should probably keep it in the dependent keys list. I realise `config` is unlikely to change dynamically at the moment but it's better to be consistent with specifying dependent keys for future compatibility and avoiding difficult to spot issues in tests.

Reply:
👍 Oops! Fixed and pushed!